### ISSUE TITLE
Add support for GtkGesture

### ISSUE THREAD

**Issue by BorisLeMeec** [gotk3/gotk3#305]
Hi, I'd like to use that function `gtk_gesture_drag_get_start_point ()`

But it's not yet implemented, can you just give some advice on how can I implement it from my side so I can use it with your library, thank you a lot.

Or if you know another way I can get the cursor position at the moment of the click, and at every moment before the release of the click.

Thank you

**Comment by BorisLeMeec** [comment]
Yes of course, by implementing it on my side I meant implement it then make a PR, but I really have no idea about where to start, that's the point

**Comment by Bios-Marcel** [comment]
@BorisLeMeec I don't really know how to lead you into the right direction. Just check out what's necessary in order to implement everything you need and see how similar stuff was done. Most bindings are pretty straightforward.
